### PR TITLE
Kill PhantomShim.

### PR DESCRIPTION
**Pull request by aslushnikov**
This patch kills PhantomShim, a lightweight layer that helped to
bootstrap Puppeteer API.

At this point, PhantomShim:
- passes 139 phantomJS tests
- fails 13 tests
- lacks capabilities to run 26

The failings and unsupported tests don't have any value for Puppeteer
API, so it doesn't make sense to maintain PhantomShim any longer.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +18,6 @@
     "coverage": "COVERAGE=true npm run unit",
     "node6": "node utils/node6-transform/index.js",
     "test-node6": "jasmine utils/node6-transform/test/test.js",
-    "build": "npm run node6",
```
Comment (JoelEinbinder) [inline package.json]: I think this was unintentional removal.

**Comment by JoelEinbinder** [comment]
Github lags when looking through the files changed, but I think you missed the entires in `.gitignore` and `.npmignore`

File: package.json (hunk 1 above)
Comment (aslushnikov) [reply package.json]: That's a drive-by. Do we need it?

**Comment by aslushnikov** [comment]
Thanks, cleaned up `.npmignore`. There's nothing related to phantom shim in `.gitignore`

File: package.json (hunk 1 above)
Comment (JoelEinbinder) [reply package.json]: I wanted to future proof, in case we have more build steps.

**Review by JoelEinbinder — APPROVED** [review]

**Pull request merged**